internal/outputgenerators/templates/cluster_settings.txt

```diff
@@ -39,14 +39,15 @@ Group.nrofApplications = 0
 Group{{add $index 1}}.groupID = {{$group.Id}}
 Group{{add $index 1}}.nrofHosts = {{$group.NrofHosts}}
 Group{{add $index 1}}.nrofInterfaces = {{$group.NrofInterfaces}}
+{{- if eq .DefaultPosition true}}
```

Comment:
Why are you using `.DefaultPosition` and not `$group.DefaultPosition` as every other parameter here?

Reply:
Also: if it's the _default_ position, it is not the user specified, isn't it?